docs/tutorial/part-two/index.md

```diff
@@ -96,238 +50,102 @@ This creates a new site with the following structure.
-
-Now, all the text font sizes should be slightly bigger. Try changing the
-`baseFontSize` to `24px` then `12px`. All elements get resized as their
-`font-size` is based on the `baseFontSize`.
+> Note: The placement of the example css file in a `/src/styles/` folder is arbitrary.
 
-_Note that if you use `gatsby-plugin-typography` with the default starter, you'll need to delete the default layout.css used by that starter as it overrides the Typography.js CSS_
+#### ✋ Include the stylesheet in `gatsby-browser.js`
 
-There are
-[many themes available](https://github.com/KyleAMathews/typography.js#published-typographyjs-themes)
-for Typography.js. Try a couple. In your terminal at the root of your
-site, run:
+1. Create the `gatsby-browser.js`
 
 ```shell
-npm install --save typography-theme-bootstrap typography-theme-lawton
+cd ../..
+touch gatsby-browser.js
 ```
 
-To use the Bootstrap theme, change your typography code to:
-
-```javascript:title=src/utils/typography.js
-import Typography from "typography"
-import bootstrapTheme from "typography-theme-bootstrap" // highlight-line
+Your project's file structure should now look like this:
 
-const typography = new Typography(bootstrapTheme) // highlight-line
-
-export default typography
+```text
+├── package.json
+├── src
+│   └── pages
+│       └── index.js
+│   └── styles
+│       └── global.css
+├── gatsby-browser.js
 ```
 
-![typography-bootstrap](typography-bootstrap.png)
+> 💡 What is `gatsby-browser.js`? Don't worry about this too much for now -- For now, know that `gatsby-browser.js` is one of a handful of special files that Gatsby looks for, and uses (if they exist). Here, the naming of the file **is** important.
 
-Themes can also add Google Fonts. The Lawton theme you installed along with the
-Bootstrap theme does this. Replace your typography module code with the
-following, then restart the dev server (necessary to load the new Google Fonts).
+2. Import your recently-created stylesheet in the `gatsby-browser.js` file:
 
-```javascript:title=src/utils/typography.js
-import Typography from "typography"
-// highlight-start
-// import bootstrapTheme from "typography-theme-bootstrap"
-import lawtonTheme from "typography-theme-lawton"
-// highlight-end
+```javascript:title=gatsby-browser.js
+import "./src/styles/global.css"
 
-const typography = new Typography(lawtonTheme) // highlight-line
-
-export default typography
+// or:
+// require('./src/styles/global.css')
 ```
 
-![typography-lawton](typography-lawton.png)
-
-_Challenge:_ Typography.js has more than 30 themes!
-[Try them live](http://kyleamathews.github.io/typography.js) or check out
-[the complete list](https://github.com/KyleAMathews/typography.js#published-typographyjs-themes) and try installing one on your current Gatsby site.
-
-## Component CSS
+> Note: You can use Node.js `require` or `import` syntax. If you're not sure of the difference, just pick one.
```

Comment:
Added a little bit of extra info and made a recommendation instead of putting the choice on the reader.

```suggestion
> Note: Both CommonJS (`require`) and ES Module (`import`) syntax work here. If you’re not sure which to choose, we use `import` most of the time.
```